docs/maps/mapml-viewer.md

```diff
@@ -0,0 +1,56 @@
+
+```html
+<!DOCTYPE html>
+<html>
+   <head>
+     <title>A Simple Web Map[tm]</title>
+     <meta charset="utf-8" />
+     <script type="module" src="web-map/mapml-viewer.js"></script>
+     <style>html {height:100%} * {margin:0px;padding:0px} body,mapml-viewer {height:inherit}</style>
+   </head>
+   <body>
+    <mapml-viewer projection="OSMTILE" zoom="0" lat="0.0" lon="0.0" controls>
+        <layer- label="OpenStreetMap" src="https://geogratis.gc.ca/mapml/en/osmtile/osm/" checked></layer->
+    </mapml-viewer>
+   </body>
+</html>    
+```
+Note that for the above example to run properly on your own site, you need to get a built copy of the `<mapml-viewer>` project in your site's folder. In the example, the `<mapml-viewer>` files are copied into the folder named "web-map" in your site root folder. Your own site's path to these files will depend on how you structure your folders.
+
+`<mapml-viewer>` is an "[autonomous custom element](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements)" in HTML.  You can distinguish a custom element of any category from a "native" HTML element by the "-" anywhere in the element name. Autonomous custom elements are supported by all modern browsers, but don't work in old browsers (e.g. Internet Explorer and old Edge).
+
+The `<mapml-viewer>` element has several attributes to control the presentation and initial location of the map.  
+
+### Attributes
+
+
+`projection` - an enumerated attribute. Case-sensitive values are: "`OSMTILE`", "`WGS84`", "`CBMTILE`" and "`APSTILE`".  
+
+  - `OSMTILE` corresponds to the widely-used "Web Mercator" projected coordinate reference system, implying a "tile pyramid" zoom range from 0 to 23 (minimum tile size ~2.4m).
+
+  - `WGS84` provides an implementation of the "pseudo plate carée" projected coordinate reference system, wherein the easting and northing axis units are decimal degrees (not meters). In `WGS84`, zoom level 0 contains two tiles that each cover a hemisphere of Earth's surface. `WGS84` is defined with 21 zoom levels (0 to 20).  
+
+  - `CBMTILE` is the de facto standard grid layout for the Canadian Geospatial Data Infrastructure (CGDI), defined by Natural Resources Canada, and is based on the Lambert Conformal Conic projection (EPSG:3978). Zoom levels are based on a numeric map scale denominator (e.g. 10000 corresponding to a map scale of 1:10,000), with a particular pixel resolution selected, and as a result, successive zoom levels' tiles do not nest exactly (as they do in `OSMTILE`, `WGS84` and `APSTILE`). 
+
+  - `APSTILE` is based on the Alaska Polar Stereographic (EPSG:5936) projected coordinate reference system, and has 20 zoom levels (0 to 19).
+
+  - other projections are possible, using the "[Custom Projections](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/239/commits/e9a29e7abcc43d7f15eb64729920e2ad06fc25c5)" API
```

Comment:
Alternatively this link could point to the file at https://github.com/Maps4HTML/Web-Map-Custom-Element/blob/master/demo/CustomProjection.html (instead of the PR commit).

Or permalink: https://github.com/Maps4HTML/Web-Map-Custom-Element/blob/8723e4d8bad8e3629bc9dca7a6fe81d724e8418f/demo/CustomProjection.html#L1-L158